test/tests/fips_test_suite/common.rs

```diff
@@ -237,11 +237,101 @@ pub fn mbx_send_and_check_resp_hdr<T: HwModel, U: FromBytes + AsBytes>(
     //Ok(U::read_from(resp_bytes.as_bytes()).unwrap())
 }
 
-// Returns true if not all bytes are the same
+fn get_cmd_id(dpe_cmd: &mut Command) -> u32 {
```

Comment:
Should be able to use `u32::from(cmd)`: https://github.com/chipsalliance/caliptra-dpe/blob/main/dpe/src/commands/mod.rs#L89-L102

Reply:
From consumes the value, which we need to stick around.

Reply:
Ack. It would be pretty trivial to also implement `From<&Command>`. Something like:

```rust
impl From<&Command> for u32 {
  fn from (c: &Command) -> u32 {
    <normal implementation>
  }
}

impl From<Command> for u32 {
  fn from (c: Command) -> u32 {
    u32::from(&c)
  }
}
```

But not needed for this PR.

Reply:
Oh, especially since the change would have to be in the DPE repo.